capstone/capapi/constants.py

```diff
@@ -0,0 +1,4 @@
+OPEN_CASE_JURISDICTIONS = [
```

Comment:
See later comments -- I suggest making `whitelisted` a field on the Jurisdiction model instead of setting this as a constant.